test-common/build.gradle

```diff
@@ -30,17 +30,5 @@ dependencies {
     errorprone 'com.google.errorprone:error_prone_core:2.3.1'
     compileOnly 'org.projectlombok:lombok:1.18.0'
 
-    compile 'org.hamcrest:java-hamcrest:2.0.0.0'
```

Comment:
I'm a bit surprised this works. The hamcrest usage is in src code for test-common, which is then packaged and included as a test dependency in other projects. Removing the compile time dependency seems like it should break, hamcrest would not be on the src path when compiling.

Reply:
@DanVanAtta It gets readded a couple of lines below.

Reply:
ah, makes sense - thank you